doc/conf.py

```diff
@@ -28,7 +30,7 @@
 
 # General information about the project.
 project = u'PyMongo'
-copyright = u'2008 - 2015, MongoDB, Inc.'
+copyright = u'2008-{0}'.format(datetime.date.today().year)
```

Comment:
Clever. Can you add back the `", MongoDB, Inc."`? Overwise the bottom of the page just says "© Copyright 2008-2017. Created using Sphinx 1.4.5."